plugins/crd/crd_queue.go

```diff
@@ -114,6 +132,7 @@ func networkservicechannelWork(plugin *Plugin) {
 
 		// If the queue has been shut down, we should exit the work queue here
 		if shutdown {
+			plugin.Log.Error("shutdown signaled, closing stopChNSC")
 			plugin.stopChNSC <- struct{}{}
```

Comment:
This takes a struct{} but a struct{}{} is passed in. Any reason for this?

Reply:
If I try to pass in a `struct{}` I get the error:

`type struct{} is not an expression`

Reply:
Now that you mention this, I do recall this came up in the code walkthrough we did a few weeks back. But lets open an issue to track changing this.